.github/actions/check-pr-status/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "check-pr-status",
-  "version": "5.4.2",
+  "version": "5.6.0",
```

Comment:
Can someone confirm this is ok? Not sure why it wouldn't have updated previously

Reply:
Also not sure why it chose to update to 5.6.0 instead of 5.7.0

Reply:
it's not the automated release, it's a change made in a PR https://github.com/strapi/strapi/pull/22346